certified-connectors/Connect2All/apiProperties.json

```diff
@@ -0,0 +1,45 @@
+{
+  "properties": {
+    "capabilities": [],
+    "connectionParameters": {
+      "token": {
+        "oAuthSettings": {
+          "clientId": "30118c48-c78c-40a1-84b7-80961849e93b",
```

Comment:
Please remove the client ID and replace it with a dummy value

Reply:
Removed the clientId and put in a dummy value.